BggAlerter/Program.cs

```diff
@@ -29,7 +29,6 @@ private static async Task RunApplication()
             // TODO: check database for existing entries, load to memory if any
             var httpResponse = await bggCaller.GetBggResponse();
             var gameList = bggResponseParser.ParseBggResponse(httpResponse);
-
             var bggFileWriter = new BggFileWriter();
```

Comment:
it is NOT resolved - move class instantiation together, See previous comments which you clicked as resolved without resolving